### PR TITLE
Docs: Fix Recommendations URL

### PR DESCRIPTION
**Pull request by acs-ferreira**

### CODE REVIEW

**Comment by queenypingcap** [comment]
LGTM. Thanks!

**Comment by ngaut** [comment]
LGTM

File: op-guide/binary-deployment.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ category: deployment
 
 A complete TiDB project contains PD, TiKV, TiDB. The start-up sequence is PD -> TiKV -> TiDB.
 
-Before your start, see [TiDB Architecture](../README.md#TiDB-Architecture) and [Deployment Recommendations](op-guide/recommendation.md).
+Before your start, see [TiDB Architecture](../README.md#TiDB-Architecture) and [Deployment Recommendations](../op-guide/recommendation.md).
```
Comment (shenli) [inline op-guide/binary-deployment.md]: Change "../op-guide/recommendation.md" to "recommendation.md".

File: op-guide/binary-deployment.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ cd tidb-latest-linux-amd64-centos6
 
 ## Multiple Nodes Cluster Deployment
 
-For production environment, multiple nodes cluster deployment is recommended. Before you begin, see [Deployment Recommendations](op-guide/recommendation.md).
+For production environment, multiple nodes cluster deployment is recommended. Before you begin, see [Deployment Recommendations](../op-guide/recommendation.md).
```
Comment (shenli) [inline op-guide/binary-deployment.md]: Ditto.

**Pull request merged**